scripts/procedures/migrations.ts

```diff
@@ -135,6 +135,16 @@ export const migration: T.ExpectedExports.migration = compat.migrations
           { version: "22.11.1", type: "down" },
         ),
       },
+      "23.02.2": {
+        up: compat.migrations.updateConfig(
+          (config) => {
+            return config;
+          },
+          true,
+          { version: "23.02.2", type: "up" },
+        ),
+        down: () => { throw new Error('Cannot downgrade') },
+      },
```

Comment:
This all definitely works. Another way to do this would be:

```suggestion
      "23.02.2": {
          up: migrationFn(async _effects => await { configured: true }),
          down: () => { throw new Error('thing') },
        },
```

However, I have not tested above so I'm not 💯 it works as expected 😺